# HG changeset patch
# User Rik <rik@octave.org>
# Date 1456010784 28800
#      Sat Feb 20 15:26:24 2016 -0800
# Node ID ea2c2e08ceda2314d6be049933038ad76103e358
# Parent  a4235a7eeb2c4b1c1bad6b4a4bb45d80024a5399
doc: Periodic spellcheck of documentation.

* aspell-octave.en.pws: Add new words to custom dictionary.

* intro.txi: Use @nospell macro around proper names.
Change notational to notation.

* poly.txi: Use Texinfo to generate accent for Pade'.
Correct peicewise to piecewise.

* cubehelix.m: Correct transversing to traversing.

* padecoef.m: Use @nospell macro around small wordlets.

* diffpara.m, spencer.m, pcr.m, quantile.m: Use @nospell macro around proper
names.
***
doc: Correct spelling of piecewise in Polynomial chapter of manual.

diff --git a/doc/interpreter/doccheck/aspell-octave.en.pws b/doc/interpreter/doccheck/aspell-octave.en.pws
--- a/doc/interpreter/doccheck/aspell-octave.en.pws
+++ b/doc/interpreter/doccheck/aspell-octave.en.pws
@@ -1,68 +1,74 @@
 personal_ws-1.1 en 1
 accumarray
+acknowledgements
 Acknowledgements
-acknowledgements
 ACM
 adams
 Affero
 afterwards
 aifm
 AIX
 al
 aleph
 alnum
 alphanum
 amd
+amongst
+anisotropic
+anova
 ANOVA
-anova
 Anymap
 API
 APIs
 appdata
 approximant
 arg
 args
 ARMA
 arpack
 ascii
+associativity
+Associativity
 ast
 async
 atan
 Attr
 audiodevinfo
 audioplayer
 audiorecorder
 AutoCAD
 Autoconf
 autocorrelations
 autocovariances
 autoload
 Autoload
 autoloaded
+Autoloading
 autoloading
-Autoloading
 Automake
 autoregression
 autoregressions
 autoscaled
 AutoScaling
 AWK
 awt
 backend
 Backends
 backends
 backgroundcolor
 backtrace
 bartlett
+Barycentric
 BaseValue
 basevalue
+bdf
 BDF
-bdf
+benchmarking
 betacdf
 betainv
 betaln
 betapdf
 betarnd
 BFGS
 BICG
 BiConjugate
@@ -91,30 +97,30 @@ brackety
 braindead
 breakpoint
 broadcastable
 BSX
 builtin
 builtins
 ButtonDownFcn
 BV
+BZ
 bz
-BZ
 bzip
 CallbackObject
 callee's
 camd
 CancelString
 cauchy
 caxis
 ccolamd
 cd
 cdata
+cdf
 CDF
-cdf
 CDFs
 cdot
 ceil
 cellstr
 ChangeLog
 changelogs
 changeset
 changesets
@@ -128,16 +134,17 @@ chisquare
 chol
 Cholesky
 cholmod
 chromaticity
 chrominance
 cindex
 circ
 circshift
+circulant
 circum
 classpath
 classpaths
 CLI
 clim
 climmode
 cloglog
 closerequestfcn
@@ -156,52 +163,59 @@ colorbar
 colormap
 colormaps
 ColorOrder
 colororder
 colperm
 Comint
 Commandline
 commentstyle
+compilable
 ComplexEqn
 cond
 condest
 conformant
 cong
 const
 contextless
 contourc
 convhull
+Convolve
+copyrightable
 Corel
 CorelDraw
 corrcoef
 cosecant
 courseware
 cov
+covariance
 CPLEX
 createcancelbtn
 CreateFcn
 CRLF
 Crout
 crout
+cryptographic
 csymamd
 ctranspose
+CTRL
 Ctrl
-CTRL
 CTS
 cubehelix
 cummax
 cummin
 cumprod
 cumsum
 cURL
 CurrentAxes
 CurrentFigure
 CurrentObject
 currentpoint
+customizable
+customizations
 cxsparse
 Cygwin
 DAE
 DAEs
 daspk
 dasrt
 dassl
 dataset
@@ -211,55 +225,64 @@ datenum
 datestr
 datestrings
 davis
 ddd
 dddd
 deallocate
 deallocated
 deconv
+Deconvolve
+decrement
 deftypefn
+delaunay
 Delaunay
-delaunay
 delaunayn
 DeleteFcn
 delim
 deltaX
 demi
 DeskJet
 det
 diag
 diagcomp
+dialogs
+Dialogs
 diamondsuit
+differentiable
 digamma
+dimensionality
+dimensionally
 dir
 disp
+displayname
 DisplayName
-displayname
 DisplayNames
 displayrange
+distributable
 dMatrix
 dmperm
 docstring
 docstrings
 dOmega
 doNotSpecify
 dotall
 dotexceptnewline
 doublearrow
+downarrow
 Downarrow
-downarrow
 downdate
 dpi
 droptol
+Dt
 dt
-Dt
 dx
 dy
 EastOutside
+Econometrics
 edgecolor
 Ei
 EIDORS
 eigenpairs
 eigenvector
 eigenvectors
 eigs
 elementwise
@@ -273,16 +296,18 @@ endif
 endofline
 eof
 EOF
 EOLs
 EOS
 eps
 eq
 equidistributed
+equilibration
+Equilibration
 equispaced
 erf
 erfc
 erfi
 errno
 Errorbar
 errorbar
 Errorbars
@@ -299,21 +324,22 @@ Exif
 exitflag
 expcdf
 expinv
 exppdf
 exprnd
 extendedtext
 facecolor
 facevertexcdata
+factorizations
 fcdf
 femmodel
 FFF
+fft
 FFT
-fft
 fftn
 fftpack
 FFTs
 fftw
 fieldname
 fieldnames
 FIFOs
 filename
@@ -374,16 +400,17 @@ gfortran
 Ghostscript
 Ghostscript's
 gif
 GIF
 glibc
 globbing
 glpk
 GLS
+glyphs
 GMRES
 gnulib
 gnuplot
 Gnuplot
 gnuplot's
 goto
 Goto
 gotos
@@ -402,18 +429,18 @@ gswin
 GUIs
 gunzipped
 gz
 gzip
 gzipped
 Hadamard
 HandleVisibility
 Hankel
+Hanning
 hanning
-Hanning
 hardcode
 hardcoding
 HDF
 hdf
 headerlines
 headlength
 headstyle
 headwidth
@@ -454,35 +481,40 @@ ifft
 ifftn
 ignorecase
 ij
 ilu
 ilutp
 Im
 imag
 ImageMagick
+indices
 Indices
-indices
+infeasible
 infty
 init
 InitialValue
+Inline
 inline
-Inline
 inpolygon
 inputdlg
+integrand
+integrands
+integrators
 internet
 interp
 interpderiv
 interpft
 interpn
 interpolant
 Interpolants
 interquartile
 inuse
 inv
+invariants
 involutory
 ipermute
 isdir
 ishandle
 ishghandle
 isolines
 isosurface
 isovalue
@@ -539,33 +571,35 @@ lfloor
 li
 libcurl
 liblapack
 liboctave
 liboctinterp
 Libtool
 licensors
 lineanchors
+linearized
 linefeeds
 linesearch
 linespec
 linespoints
 linestyle
 linewidth
 linkprop
 listbox
 listdlg
 ListSize
 ListString
 literalspacing
 LLVM
+lm
 LM
-lm
 loadpath
 Lobatto
+logarithmically
 logfile
 logit
 logncdf
 logninv
 lognormal
 lognpdf
 lognrnd
 lookup
@@ -574,49 +608,51 @@ lookups
 Lovato
 lpx
 lr
 lsode
 lt
 lu
 Lua
 luinc
+luminance
 lvalue
 lvl
 lx
 ly
 lz
 lzma
 lzw
 Mahalanobis
 makefile
 makefiles
 makeinfo
 Mandriva
+manova
 MANOVA
-manova
 markeredgecolor
 markerfacecolor
 markersize
 markerstyle
 matchcase
 matlab
 MaxFunEvals
 maxima
 MaxIntervalCount
 MaxIter
 mcnemar
 mdl
 meansq
 menubars
+MERCHANTABILITY
 Mersenne
 meshgrid
 meshgridded
+Metafile
 metafile
-Metafile
 metafiles
 Metafont
 mex
 mget
 michol
 Microsystems
 MinGW
 minima
@@ -647,18 +683,19 @@ multibyte
 multiline
 multipage
 multipledelimsasone
 MultiSelect
 mxArray
 myclass
 myfun
 nabla
+NAMESPACE
 namespace
-NAMESPACE
+nan
 NaN
 NaNs
 nargin
 nargout
 nbincdf
 nbininv
 nbinpdf
 nbinrnd
@@ -681,91 +718,101 @@ nocompute
 nodither
 nofill
 nolabel
 noncommercially
 nonconformant
 nondecreasing
 nonincreasing
 nonnan
+nonparametric
 nonsmooth
 nonsparse
 nonzeros
 NOP
 noperm
+normals
 normcdf
 normest
 norminv
 normpdf
 normrnd
 NorthOutside
 noscal
 noshare
 notin
 nrows
 nthargout
+ntsc
 NTSC
-ntsc
 numel
 Numpy
 nzmax
 oct
 octaverc
 ODEPACK
 ODEs
+ogg
 Ogg
-ogg
 OKString
 OLS
 onCleanup
 online
 OpenBLAS
 OpenGL
 OpenJDK
 oplus
+optimizations
 Ord
 oregonator
 Orthogonalize
+orthonormal
 oslash
 otimes
 outerposition
+outlier
+outliers
+outputfcn
 OutputFcn
-outputfcn
+overdetermined
 overridable
 paperorientation
 paperposition
 PaperPosition
 papersize
+PaperUnits
 paperunits
-PaperUnits
+parametrically
 parseparams
+PBM
 pbm
-PBM
 PBMplus
 pc
+PCG
 pcg
-PCG
 pchip
 pcr
+PCRE
 pcre
-PCRE
+pcx
 PCX
-pcx
+PDF
 pdf
-PDF
 pdflatex
 pearson
 pentadiagonal
 periodogram
 perp
 PGF
 pgm
 PGMRES
 PHP
 pict
+Piecewise
+piecewise
 pinv
 Pixmap
 png
 PNG
 pnm
 PointJet
 poisscdf
 poissinv
@@ -779,27 +826,29 @@ polyval
 popupmenu
 pos
 POSDEF
 POSIX
 postorder
 PostScript
 powerset
 pre
+preallocate
 preconditioner
 preconditioners
 premultiplied
 Prepend
 prepended
+preselected
 presolver
 printf
 priori
 probit
+profiler
 Profiler
-profiler
 programmatically
 prolate
 PromptString
 propto
 proven
 ps
 pseudoinverse
 pseudonorm
@@ -809,122 +858,138 @@ pseudospectrum
 ptrace
 punct
 pushbutton
 Pxx
 Qhull
 qhull
 QP
 QQ
+QRUPDATE
 qrupdate
-QRUPDATE
 QScintilla
 quadcc
 quadgk
 quadl
 quadpack
 quadv
+Quantile
 quantile
-Quantile
 quantiles
+quantization
+quartic
+quartile
 questdlg
 Quickhull
+QZ
 qz
-QZ
+radian
+radians
 radices
 radiobutton
+radix
 randn
 randperm
 rangle
 ranlib
 ras
 rceil
 rdivide
+readline
 Readline
-readline
 RECT
 recursing
+redistributable
 reentrant
 regex
 regressor
 Reindent
 relicensing
 ren
 renderer
 repelems
 replacechildren
 repmat
+representable
+reproducibility
 resampled
 resampling
 Resize
 resize
 resized
+resizing
+Resizing
 resnorm
 resparsify
 restoreBG
 restorePrevious
 RET
 returnonerror
 rfloor
+RGB
 rgb
-RGB
 Riccati
 Rightarrow
 rightarrow
 rle
 rline
 rmdir
 RMS
 rnd
 roffset
 Rosser
 rpath
 RPMs
 runtime
 sa
 SAS
+Scalable
 schar
 Schur
 SCO
 screenful
 se
 sed
 seealso
 SelectionMode
 semidefinite
 Sep
 SHA
 shiftdim
 si
+SIGNUM
 signum
-SIGNUM
 sim
 SIMAX
 SIMD
 simplechol
 simplecholperm
 simplematrix
+simplexes
 simplices
 sinc
 sinetone
+skewness
 slx
 sm
 smirnov
 Smirnov
 SouthOutside
 sp
 spadesuit
 spalloc
 SPARSKIT
 spaugment
 spchol
 spcholperm
 spconvert
 spdiags
 Spearman's
+specialness
+specular
 speye
 spfun
 splinefit
 spmatrix
 spones
 spparms
 sprand
 sprandn
@@ -932,16 +997,17 @@ sprandsym
 sprank
 SPSS
 spstats
 SQP
 sqrt
 sqrtm
 sr
 ss
+sT
 stairstep
 startup
 Startup
 statinfo
 stdin
 stdnormal
 stdout
 stepsize
@@ -962,56 +1028,62 @@ subdiagonals
 subdirectories
 subdirectory
 subexpressions
 subfunction
 Subfunction
 subfunctions
 Subfunctions
 subinterval
+subintervals
 Subintervals
-subintervals
 sublicenses
 Sublicensing
 submatrices
 submatrix
 submenu
+suboptimal
 subprocess
 subprocesses
 Subprocesses
 subsasgn
+subscripted
 Subscripted
-subscripted
 subscripting
 subseteq
 subsindex
+subspaces
+Subspaces
 subsref
 substring
 substrings
+subunits
 subwindows
 SuiteSparse
 sumsq
 SunOS
+Supercomputing
 superdiagonal
 superdiagonals
 superiorto
 superscripting
 supset
 supseteq
+surd
 SV
 SVD
 svd
 svds
+svg
 SVG
-svg
 SwitzerlandLight
 Sym
 sym
+symamd
 SYMAMD
-symamd
 symbfact
 symrcm
 Syntaxes
 tcdf
 Tcv
 terminal's
 tex
 texinfo
@@ -1055,79 +1127,84 @@ trapz
 treatasempty
 treelayout
 treeplot
 tridiagonal
 trigamma
 trimesh
 triplot
 trnd
+TrueColor
 truecolor
-TrueColor
 Tukey
 tuples
 Tx
 txi
 typedefs
 TypicalX
 ub
 UB
 uchar
 UHESS
 UI
+Uicontextmenu
 uicontextmenu
-Uicontextmenu
+uicontrol
 Uicontrol
-uicontrol
 uicontrols
 UID
 uimenu
 uint
 uipanel
 Uipanel
 uipanels
 Uipushtool
 uipushtool
 uipushtools
 uiputfile
+Uitoggletool
 uitoggletool
-Uitoggletool
+Uitoolbar
 uitoolbar
-Uitoolbar
 ulong
 Ultrix
 umfpack
 uminus
 Unary
 unary
 uncomment
 Uncomments
 unconvertible
+undirected
 unformatted
 unidcdf
 unidinv
 unidpdf
 unidrnd
 unifcdf
 unifinv
 UniformOutput
 unifpdf
 unifrnd
 unimodal
 Uninstall
 uninstalled
 univariate
 unix
+unmaintained
 unnorm
 unnormalized
+unordered
 unpadded
 unpermuted
 unpivoted
 unregister
+unscaled
 unshare
+Unshared
 unsymmetric
 untabified
 uparrow
 Uparrow
 uplus
 userdata
 Username
 ushort
@@ -1136,60 +1213,65 @@ Vandermonde
 variable's
 varphi
 varpi
 varsigma
 vartheta
 vback
 vec
 vech
+vectorization
+vectorize
 Vectorize
-vectorize
+vectorized
+Vectorized
 vectorizing
 vee
 versa
 vertcat
 VertexNormals
 verticalalignment
 vertices
+viewable
 Vorbis
 vorbis
+Voronoi
 voronoi
-Voronoi
 waitbar
 waitbars
 wallis
 warndlg
 wav
 WAV
 WayPoints
 wblcdf
 wblinv
 wblpdf
 wblrnd
 Weibull
 Welch
 welch
 WestOutside
+Whitespace
 whitespace
-Whitespace
 whos
 wienrnd
 Wikipedia
+Wilcoxon
 wilcoxon
-Wilcoxon
 wildcard
 Wildcards
 wildcards
 windowbuttondownfcn
 windowbuttonmotionfcn
 windowbuttonupfcn
 windowstyle
 WIPO
 wireframe
+workspace
 wp
 wspace
 xb
 xboxes
 xc
 xcorr
 xdata
 xdigit
@@ -1224,15 +1306,15 @@ ydata
 yerrorbar
 yerrorbars
 ylim
 ymax
 ymin
 ypos
 yy
 YY
+YYYY
 yyyy
-YYYY
 yyyymmdd
 yyyymmddTHHMMSS
 Ziggurat
 zlib
 zlim
diff --git a/doc/interpreter/intro.txi b/doc/interpreter/intro.txi
--- a/doc/interpreter/intro.txi
+++ b/doc/interpreter/intro.txi
@@ -89,17 +89,17 @@ Octave can easily be used for basic nume
 knows about arithmetic operations (+,-,*,/), exponentiation (^),
 natural logarithms/exponents (log, exp), and the trigonometric
 functions (sin, cos, @dots{}).  Moreover, Octave calculations work
 on real or imaginary numbers (i,j).  In addition, some mathematical
 constants such as the base of the natural logarithm (e) and the ratio
 of a circle's circumference to its diameter (pi) are pre-defined.
 
 @noindent
-For example, to verify Euler's Identity,
+For example, to verify @nospell{Euler's} Identity,
 @tex
 $$e^{\imath\pi} = -1$$
 @end tex
 @ifnottex
 @display
 
  i*pi
 e     = -1
@@ -418,17 +418,17 @@ The part of Octave's help facility that 
 text of the printed manual from within Octave normally uses a separate
 program called Info.  When you invoke Info you will be put into a menu
 driven program that contains the entire Octave manual.  Help for using
 Info is provided in this manual, @pxref{Getting Help}.
 
 @node Conventions
 @section Conventions
 
-This section explains the notational conventions that are used in this
+This section explains the notation conventions that are used in this
 manual.  You may want to skip this section and refer back to it later.
 
 @menu
 * Fonts::
 * Evaluation Notation::
 * Printing Notation::
 * Error Messages::
 * Format of Descriptions::
diff --git a/doc/interpreter/poly.txi b/doc/interpreter/poly.txi
--- a/doc/interpreter/poly.txi
+++ b/doc/interpreter/poly.txi
@@ -139,17 +139,17 @@ data points.  To avoid a highly fluctuat
 wants to fit a low-order polynomial to data.  This usually means that it
 is necessary to fit the polynomial in a least-squares sense, which just
 is what the @code{polyfit} function does.
 
 @DOCSTRING(polyfit)
 
 In situations where a single polynomial isn't good enough, a solution
 is to use several polynomials pieced together.  The function
-@code{splinefit} fits a peicewise polynomial (spline) to a set of
+@code{splinefit} fits a piecewise polynomial (spline) to a set of
 data.
 
 @DOCSTRING(splinefit)
 
 The number of @var{breaks} (or knots) used to construct the piecewise
 polynomial is a significant factor in suppressing the noise present in
 the input data, @var{x} and @var{y}.  This is demonstrated by the example
 below.
@@ -344,17 +344,17 @@ The result of which can be seen in @ref{
 
 @float Figure,fig:splinefit6
 @center @image{splinefit6,4in}
 @caption{Comparison of two different levels of robust fitting (@var{beta} = 0.25 and 0.75) to noisy data combined with outlying data.  A conventional fit, without
 robust fitting (@var{beta} = 0) is also included.}
 @end float
 @end ifnotinfo
 
-A very specific form of polynomial interpretation is the Pade' approximant.
+A very specific form of polynomial interpretation is the Pad@'e approximant.
 For control systems, a continuous-time delay can be modeled very simply with
 the approximant.
 
 @DOCSTRING(padecoef)
 
 The function, @code{ppval}, evaluates the piecewise polynomials, created
 by @code{mkpp} or other means, and @code{unmkpp} returns detailed
 information about the piecewise polynomial.
diff --git a/scripts/image/cubehelix.m b/scripts/image/cubehelix.m
--- a/scripts/image/cubehelix.m
+++ b/scripts/image/cubehelix.m
@@ -18,17 +18,17 @@
 
 ## -*- texinfo -*-
 ## @deftypefn  {} {@var{map} =} cubehelix ()
 ## @deftypefnx {} {@var{map} =} cubehelix (@var{n})
 ## Create cubehelix colormap.
 ##
 ## This colormap varies from black to white going though blue, green, and red
 ## tones while maintaining a monotonically increasing perception of intensity.
-## This is achieved by transversing a color cube from black to white through
+## This is achieved by traversing a color cube from black to white through
 ## a helix, hence the name cubehelix, while taking into account the perceived
 ## brightness of each channel according to the NTSC specifications from 1953.
 ##
 ## @example
 ## rgbplot (cubehelix (256))
 ## @end example
 ##
 ## The argument @var{n} must be a scalar.
diff --git a/scripts/polynomial/padecoef.m b/scripts/polynomial/padecoef.m
--- a/scripts/polynomial/padecoef.m
+++ b/scripts/polynomial/padecoef.m
@@ -26,28 +26,29 @@
 ## The Pad\'e approximant of $e^{-sT}$ is defined by the following equation
 ## $$ e^{-sT} \approx {P_n(s) \over Q_n(s)} $$
 ## where both $P_n(s)$ and $Q_n(s)$ are $N^{th}$-order rational functions
 ## defined by the following expressions
 ## $$ P_n(s)=\sum_{k=0}^N {(2N - k)!N!\over (2N)!k!(N - k)!}(-sT)^k $$
 ## $$ Q_n(s) = P_n(-s) $$
 ## @end tex
 ## @ifnottex
-## The Pad@'e approximant of exp (-sT) is defined by the following equation
+## The Pad@'e approximant of @nospell{@code{exp (-sT)}} is defined by the
+## following equation
 ##
 ## @example
 ## @group
 ##              Pn(s)
 ## exp (-sT) ~ -------
 ##              Qn(s)
 ## @end group
 ## @end example
 ##
-## Where both Pn(s) and Qn(s) are @var{N}th-order rational functions defined by
-## the following expressions
+## Where both @nospell{Pn(s) and Qn(s)} are @var{N}th-order rational functions
+## defined by the following expressions
 ##
 ## @example
 ## @group
 ##          N    (2N - k)!N!        k
 ## Pn(s) = SUM --------------- (-sT)
 ##         k=0 (2N)!k!(N - k)!
 ##
 ## Qn(s) = Pn(-s)
diff --git a/scripts/signal/diffpara.m b/scripts/signal/diffpara.m
--- a/scripts/signal/diffpara.m
+++ b/scripts/signal/diffpara.m
@@ -29,17 +29,17 @@
 ## parameter of each column is estimated.
 ##
 ## The estimators for all frequencies in the intervals described above is
 ## returned in @var{dd}.
 ##
 ## The value of @var{d} is simply the mean of @var{dd}.
 ##
 ## Reference: @nospell{P.J. Brockwell & R.A. Davis}. @cite{Time Series:
-## Theory and Methods}. Springer 1987.
+## Theory and Methods}. @nospell{Springer} 1987.
 ## @end deftypefn
 
 ## Author: FL <Friedrich.Leisch@ci.tuwien.ac.at>
 ## Description: Estimate the fractional differencing parameter
 
 function [d, dd] = diffpara (x, a, b)
 
   if (nargin < 1 || nargin > 3)
diff --git a/scripts/signal/spencer.m b/scripts/signal/spencer.m
--- a/scripts/signal/spencer.m
+++ b/scripts/signal/spencer.m
@@ -13,17 +13,18 @@
 ## General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {} {} spencer (@var{x})
-## Return Spencer's 15 point moving average of each column of @var{x}.
+## Return @nospell{Spencer's} 15 point moving average of each column of
+## @var{x}.
 ## @end deftypefn
 
 ## Author: FL <Friedrich.Leisch@ci.tuwien.ac.at>
 ## Description: Apply Spencer's 15-point MA filter
 
 function retval = spencer (x)
 
   if (nargin != 1)
diff --git a/scripts/sparse/pcr.m b/scripts/sparse/pcr.m
--- a/scripts/sparse/pcr.m
+++ b/scripts/sparse/pcr.m
@@ -160,17 +160,17 @@
 ## [x, flag, relres, iter, resvec] = ...
 ##                    pcr (A, b, [], [], "apply_m"', [], 3)
 ## @end group
 ## @end example
 ##
 ## References:
 ##
 ## [1] @nospell{W. Hackbusch}, @cite{Iterative Solution of Large Sparse
-## Systems of Equations}, section 9.5.4; Springer, 1994
+## Systems of Equations}, section 9.5.4; @nospell{Springer}, 1994
 ##
 ## @seealso{sparse, pcg}
 ## @end deftypefn
 
 ## Author: Piotr Krzyzanowski <piotr.krzyzanowski@mimuw.edu.pl>
 
 function [x, flag, relres, iter, resvec] = pcr (A, b, tol, maxit, m, x0, varargin)
 
diff --git a/scripts/statistics/base/quantile.m b/scripts/statistics/base/quantile.m
--- a/scripts/statistics/base/quantile.m
+++ b/scripts/statistics/base/quantile.m
@@ -76,17 +76,17 @@
 ## @nospell{Hyndman and Fan} (1996) recommend method 8.  Maxima, S, and R
 ## (versions prior to 2.0.0) use 7 as their default.  Minitab and SPSS
 ## use method 6.  @sc{matlab} uses method 5.
 ##
 ## References:
 ##
 ## @itemize @bullet
 ## @item @nospell{Becker, R. A., Chambers, J. M. and Wilks, A. R.} (1988)
-## The New S Language.  Wadsworth & Brooks/Cole.
+## The New S Language.  @nospell{Wadsworth & Brooks/Cole}.
 ##
 ## @item @nospell{Hyndman, R. J. and Fan, Y.} (1996) Sample quantiles in
 ## statistical packages, American Statistician, 50, 361--365.
 ##
 ## @item R: A Language and Environment for Statistical Computing;
 ## @url{http://cran.r-project.org/doc/manuals/fullrefman.pdf}.
 ## @end itemize
 ##
